src/main/java/de/retest/recheck/cli/TestReportFilterUtil.java

```diff
@@ -0,0 +1,34 @@
+package de.retest.recheck.cli;
+
+import static de.retest.recheck.ignore.RecheckIgnoreUtil.loadRecheckIgnore;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.ignore.SearchFilterFiles;
+
+public class TestReportFilterUtil {
```

Comment:
IMHO this is not a util for `TestReportFilter` (the class isn't used at all), but to filter the filters themselves based on the provided `exclude` parameter.

Reply:
Also it can be package private I guess.